<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_Screen</class>
 <widget class="QMainWindow" name="Game_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#player1, #player2{
	border: 1px solid black;
}
#randomFrame{
	border: 1px solid black;
}
#textRandom{
	text-align: center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#notification{
	background: #D3D3D3;
	color: yellow;
	font-size: 30px;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="3" column="4">
      <widget class="QFrame" name="gridFrame_6">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="gridFrame">
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QFrame" name="gridFrame_7">
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QFrame" name="gridFrame_9">
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="gridFrame_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QFrame" name="gridFrame_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QFrame" name="gridFrame_4">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QFrame" name="gridFrame_5">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QFrame" name="gridFrame_8">
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="player1">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-10</y>
      <width>161</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-image: url(:/game_bg.jpg);
}

#avatar{
	background:white;
}

#exit{
	background-color: #880808;
	border: none;
	color: yellow;
	font-size: 24px;
}

#name{
	background: none;
	color: white;
	text-align: center;
	font-size: 20px;
}

#elo{
	background: none;
	color: white;
	text-align: center;
	font-size: 15px;
}

#win{
	background: none;
	color: white;
	text-align: center;
	font-size: 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="avatar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/user_icon.qrc">
       <normaloff>:/user-solid.svg</normaloff>:/user-solid.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="elo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="win">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Win: 384 - 52%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="player2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>-10</y>
      <width>161</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-image: url(:/game_bg.jpg);
}

#avatar_2{
	background:white;
}

#name_2{
	background: none;
	color: white;
	text-align: center;
	font-size: 20px;
}

#elo_2{
	background: none;
	color: white;
	text-align: center;
	font-size: 15px;
}

#win_2{
	background: none;
	color: white;
	text-align: center;
	font-size: 15px;
}

#timer{
	background: none;
	color: white;
	text-align: center;
	font-size: 15px;
	padding-left: 20px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="avatar_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/user_icon.qrc">
       <normaloff>:/user-solid.svg</normaloff>:/user-solid.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="elo_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elo: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="win_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Win: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readyButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>390</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>READY</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="notification">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Player 1 Wins !</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="surrenderfr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #D3D3D3;
font-size: 20px;
border-radius: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>You are in game. Do you want to surrender?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="yes">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: red;
border-radius: 10px;
color: white;
font-size: 30px;</string>
     </property>
     <property name="text">
      <string>YES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="no">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: black;
border-radius: 10px;
color: white;
font-size: 30px;</string>
     </property>
     <property name="text">
      <string>NO</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource/user_icon.qrc"/>
 </resources>
 <connections/>
</ui>
